Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -193,15 +193,30 @@
     }
     @if ( editItem != null && EditMode == DataGridEditMode.Popup )
     {
-        <_DataGridModal TItem="TItem"
-                        EditItem="@editItem"
-                        EditItemCellValues="@editItemCellValues"
-                        Columns="@Columns"
-                        PopupVisible="@PopupVisible"
-                        EditState="@editState"
-                        Save="@OnSaveCommand"
-                        Cancel="@OnCancelCommand" />
-    }
+        @if ( PopupTitleTemplate == null )
```

Comment:
Don't do the `@if` statement, this can have side-effects with JSInterop used by the modal. Juzt pass the `TitleTemplate="PopupTitleTemplate"` parameter

Reply:
![grafik](https://user-images.githubusercontent.com/56487025/86495808-25bd2000-bd7b-11ea-9f18-a44415946eb0.png)
![grafik](https://user-images.githubusercontent.com/56487025/86495975-9f550e00-bd7b-11ea-8251-731fd8352feb.png)
![grafik](https://user-images.githubusercontent.com/56487025/86496009-beec3680-bd7b-11ea-8262-3d9305fdfff7.png)
I will try it. But i have problems with it. Do you know, how i can fix the excpetion?

Reply:
Check this line `@TitleTemplate( new PopupTitleContext<TItem>( EditItem, EditState ) )`. When EditItem is null the your context item is also null and you will get exception

Reply:
I think, that i have fixed it.